### PR TITLE
Update Turkish Translation

### PR DESCRIPTION
**Pull request by kyoyacchi**
updated some strings and fixed some mistranslated strings.

### CODE REVIEW

File: lib/l10n/intl_tr.arb [hunk 1]
```diff
@@ -228,12 +228,12 @@
...
   "save_painter_avatar": "Profil fotoğrafı cihaza kaydedilsin mi?",
   "favorited_tag": "En Sevilen Etiketler",
-  "legacy_mode_warning": "legacy modu Android9'dan beri kullanılamıyor",
+  "legacy_mode_warning": "Legacy modu Android 9'dan beri kullanılamıyor",
   "clear_old_format_file": "Eski biçimli dosyayı temizle",
   "clear_old_format_file_message": "_p0 olmaksızın",
   "login_error_message": "Yanlış hesap veya şifre, ya da zayıf bir şifre girdiniz, giriş başarısızlığına sebep olabilir.",
   "translate": "çevir",
-  "retry_seed_task": "Tohum görevini yeniden dene",
+  "retry_seed_task": "Tohum görevini yeniden dene(?)",
```
Comment (Notsfsssf) [inline lib/l10n/intl_tr.arb:236]: image
`(?)` Do we need to include this symbol here?

Comment (kyoyacchi) [reply lib/l10n/intl_tr.arb:236]: > image `(?)` Do we need to include this symbol here?

I couldn't understand what that option do so i put that mark there. Can you tell me what does it do? It doesn't mean any meaningful sentence in turkish

Comment (Notsfsssf) [reply lib/l10n/intl_tr.arb:236]: This option means retry all init task，I think it should be ok to remove this `(?)`?

Comment (kyoyacchi) [reply lib/l10n/intl_tr.arb:236]: > This option means retry all init task，I think it should be ok to remove this `(?)`?

Ok, i'll fix that.

**Comment by Notsfsssf** [comment]
👍

**Pull request merged**